lib/awful/keygrabber.lua

```diff
@@ -45,7 +207,411 @@ function keygrabber.stop(g)
     end
 end
 
----
+--- The keygrabber timeout.
+--
+-- @DOC_text_awful_keygrabber_timeout_EXAMPLE@
+--
+-- @property timeout
+-- @see gears.timer
+-- @see timeout_callback
+
+--- The key on which the keygrabber listen to terminate itself.
+--
+-- When this is set, the running keygrabber when quit when [one of] the release
+-- key event occurs.
```

Comment:
s/when/will/ ...and something else that I fail to figure out right now. It's too hot.